### ISSUE TITLE
Make sure settings theme works on both CM12 and 13 properly

### ISSUE THREAD

**Issue by undu** [materialos/CM12-Theme#6]
Most of the icons in the settings screen are not shown:
![Bluetooth](http://imgur.com/56X9zoG.png)
![Memory](http://imgur.com/uoqdhtN.png)
![Status Bar](http://imgur.com/cXDlSWb.png)

This happens using version 1.0 as well as the git version when using cyanogenmod 13 nightly on a motorola moto g 4g.

It also may be related that system UI crashes whenever I try to get into the Bluetooth screen, when I don't have the theme enabled it gets open just fine.

**Comment by anuragk47** [comment]
The settings icons missing on CM13 is a known bug and part of reasons why we don't say the theme works on CM 13.

**Comment by anuragk47** [comment]
@undu I will link you to a newer apk soon. Test that and tell me if the settings icons are fine there.

**Comment by anuragk47** [comment]
@undu here http://jkcloud.eu/5a012d92b8670c57

**Comment by anuragk47** [comment]
@undu also, make a separate issue for the bluetooth crash bug possibly with a log. Also keep hunting and reporting more bugs on CM 13 to help us ensure proper CM 13 support.

**Comment by undu** [comment]
@Anonymas This apk works fine, it fixed the bluetooth crash too :+1:

**Comment by undu** [comment]
On the bluetooth settings, the keyboard icon are not themed properly, they don't foloow the same placing nor color as the rest of icons:

![Bluetooth settings](http://i.imgur.com/pUciNGl.png)

**Comment by anuragk47** [comment]
@undu Please show a screenshot of the settings app with the new apk(I want to check something). We will add the keyboard icon soon.

**Comment by undu** [comment]
Have a look: [http://imgur.com/a/kL415](http://imgur.com/a/kL415)

**Comment by anuragk47** [comment]
Note to self: To close this issue:
1. Add the keyboard icon
2. Fix the dividers not being transparent.

**Comment by anuragk47** [comment]
@undu okay, btw did you find any more bugs or problems with the theme on CM 13?

**Comment by undu** [comment]
No, I'll post here if I find anything else.

**Comment by anuragk47** [comment]
Much of this is fixed by commit  76ac0db. Now only the keyboard icon and divider to close this issue.

**Comment by undu** [comment]
I've found that the settings app crashes when trying to open the battery submenu:

```
12-09 12:10:51.279 12130-12130/com.android.settings D/SubSettings: Launching fragment com.android.settings.fuelgauge.PowerUsageSummary
12-09 12:10:51.411 12130-12130/com.android.settings W/ResourceType: No package identifier when getting value for resource number 0x00000000
12-09 12:10:51.412 12130-12130/com.android.settings D/AndroidRuntime: Shutting down VM
12-09 12:10:51.414 12130-12130/com.android.settings E/AndroidRuntime: FATAL EXCEPTION: main
                                                                      Process: com.android.settings, PID: 12130
                                                                      java.lang.RuntimeException: Unable to resume activity {com.android.settings/com.android.settings.SubSettings}: android.content.res.Resources$NotFoundException: Resource ID #0x0
```

The rest of the submenus work fine for me.

**Comment by anuragk47** [comment]
Okay, this is a critical bug. We must fix this before we can release any new update.

**Comment by anuragk47** [comment]
I will just close this issue and make a new one for battery FC. Regarding the keyboard icon that isn't a major bug and will be fixed later.
